packages/cli/src/commands/server/runServer.js

```diff
@@ -62,7 +62,12 @@ async function runServer(argv: Array<string>, ctx: ContextT, args: Args) {
     watchFolders: metroConfig.watchFolders,
   });
 
-  middlewareManager.getConnectInstance().use(morgan('combined'));
+  middlewareManager.getConnectInstance().use(
+    morgan('combined'),
+    !args.verbose && {
```

Comment:
this should be a second argument to `morgan`.
Also, I've spotted and issue with `--verbose` flag being omitted because we already declare it globally. Need to investigate before merging this, so please bare with us :)

Reply:
@thymikee updated. I'll run eslint locally to avoid simple errors like this in the future. thanks

Reply:
Please use `logger.isVerbose()` for now, instead of `args.verbose`